hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/localizer/ContainerLocalizer.java

```diff
@@ -102,6 +102,9 @@ public class ContainerLocalizer {
   private static final FsPermission USERCACHE_FOLDER_PERMS =
       new FsPermission((short) 0755);
   public static final String CSI_VOLIUME_MOUNTS_ROOT = "csivolumes";
+  private static final String ADDITIONAL_JDK17_PLUS_OPTIONS =
```

Comment:
I can't see "--add-opens=java.base/java.lang=ALL-UNNAMED " here whereas I can see it in ContainerLaunch.java?
Can you explain with a code comment why is this difference?

Reply:
Added a comment on it, the need for add-exports for the sun packages here are discovered when running spark jobs on Isilon+JDK17 cluster.

Reply:
thanks!